.eslintignore

```diff
@@ -13,8 +13,12 @@
 # misc
 /coverage/
 !.*
+.eslintcache
 
 # ember-try
 /.node_modules.ember-try/
 /bower.json.ember-try
 /package.json.ember-try
+
+# ember-cli-mirage
+/mirage/
```

Comment:
Without this line, `yarn lint:js` (which runs `eslint .`) would stall locally and in CI. There may be a fix in the future, so let's look out for one.

https://www.ember-cli-mirage.com/docs/getting-started/installation#note-for-prettier-users